frontend/src/metabase/admin/utils.js

```diff
@@ -41,3 +42,18 @@ export const RedirectToAllowedSettings = connect(
   mapStateToProps,
   mapDispatchToProps,
 )(_RedirectToAllowedSettings);
+
+export const createTenantsRouteGuard = () => {
+  const Wrapper = connectedReduxRedirect({
+    wrapperDisplayName: "CanAccessTenants",
+    redirectPath: "/admin/people",
+    allowRedirectBack: false,
+    authenticatedSelector: (state) =>
+      getAdminPaths(state)?.find((path) => path.key === "people") != null &&
+      MetabaseSettings.get("use-tenants"),
```

Comment:
😭😭😭😭 please don't use MetabaseSettings unless you have to use the window object before we have a redux store loaded. I think you should be able to use `getSetting` selector here